lib/renderMap.js

```diff
@@ -335,6 +335,7 @@ function renderMap(req, res, next, apiPathEntryPoint, apiConfigUrl) {
 						mapData.iframeSrc = utils.responseUrl(req, apiPathEntryPoint + 'render/', mapId);
 						mapData.hideAttr = typeof(req.query.hideAttr) !== 'undefined';
 						mapData.cacheBuster = config.getCachebusterUrlPart();
+						mapData.isEmbeddedInWikiaApp = typeof(req.query.isEmbeddedInWikiaApp) !== 'undefined';
```

Comment:
So `&isEmbeddedInWikiaApp=false` will set `isEmbeddedInWikiaApp = true`? :)
I can see that `hideAttr` works the same way but I don't like it anyway. Your call.